app/controllers/concerns/project_scoped.rb

```diff
@@ -2,27 +2,34 @@ module ProjectScoped
   extend ActiveSupport::Concern
 
   included do
-    before_action :set_project
+    # Use prepend_before_action to keep things consistent with Pro:
+    prepend_before_action :set_project
```

Comment:
why do we need to `prepend`?

Reply:
In Pro it's necessary because, as explained in comments in that repo, `set_project` must be run before PaperTrail calls `info_for_paper_trail` (which happens in a before_action.)

In CE it doesn't make a difference, but I figured it's worth keeping things the same as Pro for consistency and future-proofing.